src/cljs/villagebookUI/components/signup.cljs

```diff
@@ -49,6 +50,7 @@
                           (:user @formdata)
                           (fn [res]
                             (swap! error assoc :message "")
+                            (user-api/get-data!)
```

Comment:
The on-click fn here is the same as that in `login.cljs`. 
Maybe you can extract these out into one.

Reply:
Yes this bit needs some refactoring. I'm planning on doing that in a separate PR in #14 .